lib/Sass/global/_normalize.scss

```diff
@@ -1,152 +1,125 @@
-/*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
+
+/* Document
+   ========================================================================== */
```

Comment:
Good catch!

I decided to upgrade normalize.css as the styling for some of the elements like `details` and `summary` didn't match the standard behavior in HTML. Newer versions don't have this problem.

I have fixed the styling changes you have pointed out. My opinion is we should keep the upgrade and fix any other inconsistencies we come across. Let me know if you think otherwise.

Reply:
Yep totally agree - we are also going to encounter subtle styling changes with the new TSX workbench PR